x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/utils/utils.ts

```diff
@@ -384,3 +385,17 @@ export const convertAlertSuppressionToSnake = (
         missing_fields_strategy: input.missingFieldsStrategy,
       }
     : undefined;
+
+export const findDifferenceInArrays = <T1, T2>(
```

Comment:
Looks like you're finding elements that are not common in either array. If yes, then consider renaming this to `findSymmetricDifferenceInArrays`

Reply:
https://github.com/elastic/kibana/pull/157966 changed the approach in here, hope you like this one better @ashokaditya  :)